src/test/java/com/google/devtools/build/lib/skyframe/FilesystemValueCheckerTest.java

```diff
@@ -756,7 +759,7 @@ private ActionExecutionValue actionValue(Action action, boolean forceDigest) {
 
   private ActionExecutionValue actionValueWithEmptyDirectory(Artifact emptyDir) {
     TreeArtifactValue emptyValue = TreeArtifactValue.create
-        (ImmutableMap.<TreeFileArtifact, FileArtifactValue>of());
+        (ImmutableSortedMap.<TreeFileArtifact, FileArtifactValue>of());
```

Comment:
nit: are the type specifications within <> necessary?

Reply:
They are not, but within the file `Immutable(Sorted)Map` is always called like this. Do you really want me to change that? If so, wouldn't it be better to do this in another PR / commit and then clean the whole file?

Reply:
This isn't any kind of convention, it's just left over from Java 7-compatibility days. I prefer incremental cleanups for this kind of thing: it keeps the blame history easier to read. So please remove the unnecessary type arguments since you're touching this line anyway. Thanks!

Reply:
Done :)